### PR TITLE
Add awsctl get vpc command #11

### PR DESCRIPTION
**Pull request by gawandepratik022**
## Fixes Issue

#11

### CODE REVIEW

**Comment by riteshsonawane1372** [comment]
@gawandepratik022 looks good.

**Comment by surajnarwade** [comment]
@gawandepratik022 I would recommend installing go lint plugin in IDE, reference: https://code.visualstudio.com/docs/languages/go

File: cmd/vpc.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+var vpcCmd = &cobra.Command{
+	Use:   "vpc",
+	Short: "Get VPCs",
+	Long:  "Get Vpcs with Name and Cidr",
+	RunE:  getVPC,
+}
+
+func getVPC(cmd *cobra.Command, args []string) error {
+
+	getVpc,err := VPC.GetVPC()
```
Comment (surajnarwade) [inline cmd/vpc.go:22]: can rename this variable to `vpcList`

Comment (surajnarwade) [reply cmd/vpc.go:22]: also gofmt and golint here

File: pkg/VPC/getVPC.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+					VpcID:    aws.String(*i.VpcId),
+				})
+			}
+
+		}
+	}
+
+	return getVpc, err
+
+}
```
Comment (surajnarwade) [inline pkg/VPC/getVPC.go:53]: newline at the end

File: cmd/vpc.go [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	vpclist, err := vpc.GetVPC()
+	if err != nil {
+		log.Fatal(err)
+	}
+	w := tabwriter.NewWriter(os.Stdout, 18, 5, 2, ' ', tabwriter.TabIndent)
+	defer w.Flush()
+
+	fmt.Fprintln(w, "NAME", "\t", "State", "\t", "DEFAULT", "\t", "IPv4CIDR", "\t", "VPCID")
+	for _, i := range vpclist {
+		fmt.Fprintln(w,
```
Comment (surajnarwade) [inline cmd/vpc.go:31]: can we use `aws.tostring` here

**Comment by surajnarwade** [comment]
can you show how output for this command looks like?

**Comment by gawandepratik022** [comment]
op

File: pkg/VPC/getVPC.go [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+					Ipv4Cidr: aws.String(*i.CidrBlock),
+					Default:  *i.IsDefault,
+					VpcID:    aws.String(*i.VpcId),
+				})
+			}
+
+		}
+	}
+
+	return vpcList, er
```
Comment (surajnarwade) [inline pkg/VPC/getVPC.go:51]: `err` ?

File: cmd/vpc.go [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+func getVPC(cmd *cobra.Command, args []string) error {
+
+	vpclist, err := vpc.GetVPC()
+	if err != nil {
+		log.Fatal(err)
+	}
+	w := tabwriter.NewWriter(os.Stdout, 18, 5, 2, ' ', tabwriter.TabIndent)
+	defer w.Flush()
+
+	fmt.Fprintln(w, "NAME", "\t", "State", "\t", "DEFAULT", "\t", "IPv4CIDR", "\t", "VPCID")
```
Comment (surajnarwade) [inline cmd/vpc.go:30]: capitalise `State`

Comment (surajnarwade) [reply cmd/vpc.go:30]: STATE

**Pull request merged**